modules/virt-attaching-vm-to-secondary-udn.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-secondary-udn.adoc
+
+:_mod-docs-content-type: PROCEDURE                                
+[id="virt-attaching-vm-to-secondary-udn_{context}"]                                  
+= Attaching a virtual machine to secondary user-defined networks by using the CLI
+
+You can connect a virtual machine (VM) to multiple secondary user-defined networks (UDNs) by requesting the pod network attachment, and configuring the interface binding.
+
+.Prerequisites
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+. Edit the `VirtualMachine` manifest to add the UDN interface details, as in the following example:
++
+.Example `VirtualMachine` manifest
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: example-vm
+  namespace: my-namespace # <1>
+spec:
+  template:
+    spec:
+      domain:
+        devices:
+          interfaces:
+            - name: cudn-localnet # <2>
+              binding: 
+                name: l2bridge # <3>
```

Comment:
this is for primary UDN. IIUC, we're describing attaching a secondary network, right ?

Reply:
Like I mentioned in my Jira comment, I have reused the code snippet from the primary UDN docs, mainly as a placeholder. Can you please provide the correct `VirtualMachine` configuration YAML snippet for secondary UDNs?